tests/integration/test_s3.py

```diff
@@ -739,7 +744,8 @@ def test_verify_can_switch_sigv4(self):
         service = self.session.get_service('s3')
         endpoint = service.get_endpoint('us-east-1')
         # The set_config_variable should ensure that we use sigv4 for s3.
-        self.assertIsInstance(endpoint.auth, botocore.auth.S3SigV4Auth)
+        # TODO: How do we check this? It's not exposed anywhere anymore...
+        #self.assertIsInstance(endpoint.auth, botocore.auth.S3SigV4Auth)
```

Comment:
Is this still an issue?  Are you just saying that this is technically supported, we just have no way to actually verify this anymore?  If so we should probably get a test in place.  We want to ensure we don't regress on this.